modules/users/client/views/employees/list-referrals.client.view.html

```diff
@@ -0,0 +1,24 @@
+<section ng-controller="ReferralsController" ng-init="find()">
+  <div class="page-header">
+    <h4>Current Referral Listings</h4>
+  </div>
+
+</br>
+  <div class="list-group">
+    <a ng-repeat="referral in referrals | filter: displayReferral" ui-sref="referrals.view({referralId: referral._id})" class="list-group-item">
+      <p class="list-group-item-text pull-right large bold">
+        <span>{{referral.title}}</span>
+      </p>
+      <p class="list-group-item-text pull-right small">
+        <span>Description: {{referral.content}}</span>
+      </p>
+      <p class="list-group-item-text pull-right small">
+        <span>Assigned to: {{referral.assigned}}</span>
+      </p>
+      -----------------------------------------------------------------
+    </a>
+  </div>
+  <div class="alert alert-warning text-center" ng-if="referrals.$resolved && !referrals.length">
+    None
+  </div>
+</section>
```

Comment:
This doesn't really look uniform with our other lists - we can fix this in another PR but it should probably look like the 'users' list [I'm leaning towards that over 'jobs' because this one is internal, but maybe we should think about making them all uniform?]